### ISSUE TITLE
mock install git fails

### ISSUE THREAD

**Issue by ykne** [rpm-software-management/mock#520]
### Short description of the problem
unable to install git in mock

### Output of `rpm -q mock`

mock-2.0-2.fc31.noarch

### Steps to reproduce issue

$ mock --init
$ mock install git
.....
INFO: installing package(s): **/builddir/git**
.....
No package **/builddir/git** available.

All other required for my project packages I tried in install line were installed correctly.
Downgraded to mock-1.4.20-1.fc31.noarch and git was installed.

**Comment by Conan-Kudo** [comment]
Did you try `mock --install git`?

**Comment by ykne** [comment]
Just tried now, same error.
```
mock --install git
...
INFO: installing package(s): /builddir/git
....
No match for argument: /builddir/git
Error: Unable to find a match: /builddir/git
```

**Comment by praiskup** [comment]
I suppose you have `git` directory in CWD;  this is going to be resolved in next version v2.1,
I'm working on the release.  For more info #475.

**Comment by praiskup** [comment]
Thank you for the report anyway!
